spk/mtproxy/src/service-setup.sh

```diff
@@ -0,0 +1,60 @@
+CFG_FILE="${SYNOPKG_PKGDEST}/var/mtproxy.conf"
+BIN="${SYNOPKG_PKGDEST}/bin/mtproto-proxy"
+AES_PWD_FILE="${SYNOPKG_PKGDEST}/var/secret.conf"
+RULES_FILE="${SYNOPKG_PKGDEST}/var/rules.conf"
+
+if [ -r "${CFG_FILE}" ]; then
+    . "${CFG_FILE}"
+fi
+
+SECRETS_CMD=$(echo "-S $PROXY_SECRETS" | sed "s|,| -S |g")
+INTERNAL_IP="$(ip -4 route get 8.8.8.8 | grep '^8\.8\.8\.8\s' | grep -Po 'src\s+\d+\.\d+\.\d+\.\d+' | awk '{print $2}')"
+EXTERNAL_IP="$(curl -s -4 "https://digitalresistance.dog/myIp")"
```

Comment:
But it can be problem if you are using dynamic IP.
This code copied from official MTProxy docker image.

Reply:
I agree with @Safihre. Asking for the user to choose the IP address is better than getting the IP from somewhere outside of the network.

As for my personal use besides `ip a s` or `ifconfig` I prefer `dig +short myip.opendns.com @resolver1.opendns.com` as It much faster (35 ms as opposed to 2.63 sec) and is backed by a large organisation so the service is likely to stick around for much longer than some random website.

The internal IP can be obtained more simply with `ip -4 route get 1 | awk '{print $NF;exit}'`

Reply:
@publicarray protocol needs **external** IP address. You can't get it by ifconfig.

Reply:
Sorry yea behind a NAT I guess not 😄

Reply:
I feel more comfortable if we used `opendns.com` or another larger service that has worldwide coverage (like a CDN or anycast DNS) for speed and availability reasons. 
In my experience some of these "find my IP" services shut down over time, are slow to respond, timeout and some organisation networks block them as they are commonly found in blocklists. If that change was made I feel OK with this. And please print an error to the log file when the curl/dig command fails. It's a single point of failure, Sorry for the rant but too often I've seen stuff break because of these hard-coded urls.

Reply:
@publicarray This code copied from official MTProto docker image. But ok, I will add alternative server.

Reply:
How about https://checkip.amazonaws.com/ ?

Reply:
The official docker image has checks in place and exits when an IP is not found. https://github.com/alexdoesh/mtproxy/blob/ef4a42b8aa7c8ed1424215c6a1b005172b9f4259/docker-entrypoint.sh#L77-L80

I think because something is official doesn't mean It cannot be improved upon.

Reply:
I added IP addresses to the wizard.
@ymartin59 please check and merge this PR